mesonbuild/compilers/swift.py

```diff
@@ -85,6 +85,12 @@ def get_include_args(self, path: str, is_system: bool) -> T.List[str]:
     def get_compile_only_args(self) -> T.List[str]:
         return ['-c']
 
+    def get_cxx_interoperability_args(self, lang: T.Dict[str, Compiler]) -> T.List[str]:
```

Comment:
This should be implemented on the base `Compiler` with an implementation that errors on usage.

Reply:
Done